gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/pages/SettingsPage.java

```diff
@@ -229,6 +230,7 @@ public void onSubmit(final AjaxRequestTarget target, final Form f) {
 			@Override
 			public void onError(final AjaxRequestTarget target, final Form<?> form) {
 				target.add(SettingsPage.this.feedbackPanel);
+				target.appendJavaScript("scroll(0,0);");
```

Comment:
Is this to scroll to the top and see the error? If so, a comment would be good here.

Reply:
Hi @adrianfish 

All changes done. Thanks!